src/script/interpreter.cpp

```diff
@@ -1297,9 +1297,12 @@ void PrecomputedTransactionData::Init(const T& txTo)
 
     // Cache is calculated only for transactions with witness
     if (txTo.HasWitness()) {
-        hashPrevouts = GetPrevoutHash(txTo);
-        hashSequence = GetSequenceHash(txTo);
-        hashOutputs = GetOutputsHash(txTo);
+        hashPrevouts = GetPrevoutsSHA256(txTo);
```

Comment:
Why this weird double assignment? `= SHA256Uint256(GetPrevoutsSHA256(txTo))` is shorter and more clear IMO.

Reply:
This PR was made before https://github.com/bitcoin/bitcoin/pull/17977#discussion_r461316301 was added (or before I saw it at least).

I specifically made it so it would be easy on rebase to switch the first assignment and argument to SHA256Uint256 to `m_hash_prevouts` for taproot, but now since that logic is all different it's not clear that this is an easier rebase. In any case, this part of the patch was designed so that master is correct, but was intended to be blown out by #17977.

Reply:
Don't worry about such trivial rebasing impact on #17977.